qemu/tests/cfg/netkvm_params_check.cfg

```diff
@@ -0,0 +1,12 @@
+- netkvm_params_check:
+    virt_test_type = qemu
+    type = netkvm_params_check
+    only Windows
+    only virtio_net
+    vhost = on
+    timeout = 360
+    smp ~= ${vcpu_maxcpus}
+    queues = ${smp}
+    netkvmco_name = "TXcapacity"
+    netkvmco_value = "64"
+    expected_log_msg = "Limit m_SGTableCapacity by 64"
```

Comment:
Update our offline discussion results here.
If we need virtio iso during testing, we need add `cdroms += " virtio"`

Reply:
@leidwang Yeah, I have added the `cdroms = " virtio cd1"` to our cfg file. It works well on my side.